setup.py

```diff
@@ -426,7 +425,8 @@ def check_file_at_path(path2file):
     setup_requires=build_requires,
     install_requires=install_requires,
     python_requires=">={}".format(min_python_version),
-    license="BSD",
+    license="MIT AND 0BSD AND BSD-2-Clause AND BSD-3-Clause AND Python-2.0",
```

Comment:
Will it help to keep a comment here to indicate which license is coming from which part?

Or maybe even as:

```
appdir_license = "MIT"
networkx_license = "MIT
cpython_license = "Python 2.0"
license_set = {appdir_license, networkx_license, cpython_license}
license = ' AND '.join(license_set)
```